lib/permission.js

```diff
@@ -269,9 +268,13 @@ api.createPermissionTable = function(resourceRoles, callback) {
     }
     // TODO: implement this as a database query
     var roles = {};
-    results.forEach(function(role) {
-      roles[role.id] = role;
-    });
+    results
+      .filter(role => {
+        return role.sysStatus !== 'deleted';
+      })
+      .forEach(role => {
+        roles[role.id] = role;
+      });
```

Comment:
@mattcollier -- Please shorten up according to @davidlehn's suggestions here and then LGTM, merge.